lib/class-wp-templates-registry.php

```diff
@@ -0,0 +1,269 @@
+<?php
```

Comment:
Should we move this class to 6.7 folder.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/61577/commits/c9fdab88f92091dfc6b0eee895498e872c1c1b7b. :slightly_smiling_face: